src/components/NewLocationPage/AboveTheFold/AboveTheFold.style.tsx

```diff
@@ -19,41 +23,58 @@ export const HeaderContainer = styled.div`
   }
 `;
 
-export const GridContainer = styled.div`
+export const GridContainer = styled.div<{ showNote: boolean }>`
   display: grid;
   max-width: ${maxContentWidth};
   margin: auto;
   row-gap: 1.25rem;
-  grid-template-areas: 'header' 'overview' 'spark' 'map' 'note' 'alerts';
+  grid-template-areas: ${({ showNote }) =>
+    showNote
```

Comment:
we need to hide the note and remove it from grid-template areas to avoid an empty grid item